### ISSUE TITLE
Missing Icons in ActionColumn with Bootstrap 4.3.x

### ISSUE THREAD

**Issue by demonking** [yiisoft/yii2-bootstrap4#162]
With the Version 4 of Bootstrap, the icons were removed from the default library.

https://getbootstrap.com/docs/4.0/extend/icons/

### What steps will reproduce the problem?

AppAsset: 

~~~
            'yii\bootstrap4\BootstrapAsset',
            'yii\bootstrap4\BootstrapPluginAsset',
~~~

GridView: 
~~~
 
   <?= GridView::widget([
        'dataProvider' => $dataProvider,
        'columns' => [
            ['class' => 'yii\grid\SerialColumn'],

            'id',
            'name',
            'description:ntext',
            'image',

            [
                'class' => 'yii\grid\ActionColumn',
            ]
        ],
    ]); ?>


~~~
### What's expected?

Seeing the icons in the last column of the gridview.

### What do you get instead?

The css classes are not present in boostrap, so the icon's as not visible.

### Additional info

| Q                | A
| ---------------| ---
| Yii version   | 2
| PHP version  | 7.2
| Operating system | Linux Debian Stretch

For the Moment I use the awesome font library and a function which creates the Buttons:

~~~
        function createActionButtons($buttons=['view','update','delete']){
                $actionButtons = [];
                $actionButtons['update']= function($url,$model) {
                         return Html::a('<i class="fa fa-edit"></i>', $url, [
                                 'title' => Yii::t('app', 'edit')
                         ]);
                };
                $actionButtons['delete'] = function($url,$model) {
                         return Html::a('<i class="fas fa-trash"></i>', $url, [
                                 'title' => Yii::t('app', 'delete')
                         ]);
                };
                $actionButtons['view'] = function($url,$model) {
                         return Html::a('<i class="fas fa-eye"></i>', $url, [
                                 'title' => Yii::t('app', 'view')
                         ]);
                };
                return array_intersect_key($actionButtons,array_flip($buttons));
        }
~~~

**Comment by samdark** [comment]
That is already mentioned in upgrade guide: https://github.com/yiisoft/yii2-bootstrap4/blob/master/docs/guide/migrating-yii2-bootstrap.md#basehtml

**Comment by samdark** [comment]
How would you fix it?

**Comment by simialbi** [comment]
Possible solution is using utf-8 Symbols instead of glyphicons in [yii2](https://github.com/yiisoft/yii2) grid view like this:
https://www.utf8icons.com/character/128065/eye
https://www.utf8icons.com/character/128393/lower-left-pencil
https://www.utf8icons.com/character/128465/wastebasket

Another alternative would be using the svg's from font awesome 5 free and mention them somewhere in the widget. They're downloadable:
https://fontawesome.com/icons/eye?style=solid

**Comment by samdark** [comment]
How would you do that? Via overwriting widget factory configs?

**Comment by simialbi** [comment]
No. In core framework itself. I would change https://github.com/yiisoft/yii2/blob/master/framework/grid/ActionColumn.php#L182 wich is bootstrap 3 specific and that's not so pretty

**Comment by samdark** [comment]
How would that work with bootstrap 3 then?

**Comment by simialbi** [comment]
the same way, inline svg of fa 5 or utf8 symbols... no glyphicons any more. If someone want to keep the glyphicons, he can override via container config

**Comment by samdark** [comment]
Is that possible to make it look the same as it is now this way?

**Comment by simialbi** [comment]
Not exactly, but with fa 5 icons nearby. Using this icons:
https://fontawesome.com/icons/eye?style=solid
https://fontawesome.com/icons/pencil-alt?style=solid
https://fontawesome.com/icons/trash-alt?style=solid

**Comment by samdark** [comment]
Then it's an option...

**Comment by simialbi** [comment]
But I think you should move it to yii2. It's not a yii2-bootstrap or a yii2-bootstrap4 problem, is it?
